website/content/docs/operations/session-recordings/index.mdx

```diff
@@ -16,6 +16,12 @@ An administrator can later view the recordings to investigate security issues, r
 Recorded sessions are stored in an external storage bucket that you create.
 Storing session recordings in a system external to Boundary means those recordings can be accessed, modified, deleted, and even restored independently of Boundary.
 You can view any sessions that Boundary recorded in your storage provider or via the CLI.
+By default, Boundary converts recorded sessions to an asciicast format for playback.
+
+Some SSH connection options are not captured by the asciicast playback format.
+If you use remote commands or an interactive shell, for example, these might not be viewable in the asciicast recorded output.
```

Comment:
This isn't quite correct. An interactive shell will be viewable as an asciicast. Perhaps something like:

```
The asciicast format is well suited for playback of interactive shells.
However, some aspects of the recording cannot be translated into the asciicast.
For example, if an SSH session uses the RemoteCommand option,
or is used to exec a command,
the command will not be displayed in the asciicast,
although the output of the command may be displayed.
If SSH is used for something other than an interactive shell,
such as for a file transfer, X11 forwarding, or port forwarding,
boundary will not attempt to create an asciicast.
In all cases the SSH session is still recorded in the [BSR files](/boundary/docs/concepts/auditing/#bsr-directory-structure).

Reply:
Thanks for the clarification, Tim! This makes sense to me. I updated the text here and in the Note on the other page.